pkg/operator/status.go

```diff
@@ -81,6 +81,21 @@ func (c *Controller) getOrCreateOperatorStatus() (*configv1.ClusterOperator, err
 	return co, nil
 }
 
+// profileApplied returns true if Tuned Profiles 'profile' has been applied.
```

Comment:
```suggestion
// profileApplied returns true if Tuned Profile "profile" has been applied.
```
Sorry I didn't catch this in the first review. After this, the PR looks good to me.

Reply:
Thank you.  Fixed in other place too.  Using `'` to quote variable names elsewhere in function comments (e.g. `mc.go`, `profilecalculator.go`), so switching to that for consistency.